frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -39,6 +39,42 @@ export const validationSchema = yup.object().shape({
         message: 'This field cannot be empty.',
       }),
   }),
+  serverless: yup.object().shape({
+    enabled: yup.boolean(),
+    scaling: yup.object().when('enabled', {
+      is: true,
+      then: yup.object({
+        minpods: yup
+          .number()
+          .integer('Pods must be an Integer.')
+          .min(0, 'Pods must be greater than or equal to 0.')
+          .test({
+            name: 'isEmpty',
+            test: (value) => value !== undefined,
+            message: 'This field cannot be empty.',
+          }),
+        maxpods: yup
+          .number()
+          .integer('Pods must be an Integer.')
+          .min(1, 'Pods must be greater than or equal to 1.')
+          .test({
+            test(limit) {
+              const { minpods } = this.parent;
+              return limit >= minpods;
+            },
+            message: 'Max Pod count must be greater than or equal Min Pod Count.',
```

Comment:
You use the word `count` here but not in the other messages.
Please confirm messages with @parvathyvr

Reply:
@invincibleJai for this message,remove the word `count`